cost-model.md

```diff
@@ -40,11 +40,6 @@ The preceding example is simplified and only used to explain the basic principle
 
 ## Cost Model Version 2
 
-> **Warning:**
```

Comment:
Add `Switching the version of the cost model might cause changes to query plans.` below Line 45.